test/integration/job/job_test.go

```diff
@@ -424,17 +424,20 @@ func TestJobPodFailurePolicy(t *testing.T) {
 			},
 		},
 	}
+
+	i := 0
+	closeFn, restConfig, clientSet, ns := setup(t, "simple")
```

Comment:
```suggestion
	closeFn, restConfig, clientSet, ns := setup(t, "pod-failure-policy")
```
nit, but as we already touch this line it makes sense to adjust. I guess it might be helpful for debugging when reading logs.